website/content/project/policies.md

```diff
@@ -0,0 +1,331 @@
+
+This may seem redundant, but take into account that closing an issue may not necessarily mean that the task is done. For example, the following could also be valid closing statements for Task #1:
+
+
+
+*   “We ultimately decided we don’t need a circle module after all. I’m closing this issue, but will reopen if we decide to revisit this”
+*   “The circle module has been implemented, except for the intersection function, which turned out to be more complex than anticipated. I am closing this issue, and have created a separate task (#5) for implementing circle intersections”
+
+
+### Pull Requests
+
+Like in many software projects, the `master` branch of our Git repository should only contain stable code that compiles, runs, and passes all tests. This is enforced by making the master branch a _protected_ branch, meaning that only the instructors and TAs have the ability to directly push to that branch. 
+
+On top of that, there is an additional `dev` (short for "development") branch that acts as a "pre-master" branch. The `dev` branch is also protected, and should only contain stable code, but it is used as a staging area to integrate code (and uncover potential integration issues) before publishing code to `master`.
+
+This means that, to get code into chiventure, you must make pull request to get code merged into the `dev` branch (and the instructor and TAs will decide when to merge that code into `master`).
+
+Because the `dev` branch should contain stable code, pull requests to master must be carefully reviewed before they are approved. If you want to merge code into `dev`, your code must be reviewed by (1) someone in your team and (2) your senior developer. When your code affects code owned by another team, you must have an additional reviewer from each team who's code is affected.
+
+A pull request should contain a relatively self-contained piece of work.  In fact, it is not uncommon for a task (or, at most, 2-3 tasks) to result in a pull request. We encourage you to touch base with your senior developer before creating your first pull requests, to determine whether the work you want to merge merits a pull request (or, similarly, whether it needs to be broken up into multiple pull requests).
+
+Like a task, a pull request should have a descriptive title, a descriptive summary, updates on the progress of the pull request (especially since changes may need to be done based on the code reviews), and a closing statement. The assignees for the pull request must be everyone who contributed to the code that is being merged. If a pull request is approved, all assignees and reviewers will receive the points for that pull request.
+
+Take into account that the final outcome of a pull request could be for that pull request to be denied. If this happens, it will not impact your score (but you will also not receive points for that pull request).
```

Comment:
maybe: "If this happens, it will not impact your score (the effect will be the same as if you had never made the pull request to begin with)" ?

Reply:
Fixed